vertx-template-engines/vertx-web-templ-mvel/src/main/java/io/vertx/ext/web/templ/impl/MVELTemplateEngineImpl.java

```diff
@@ -67,11 +70,16 @@ public void render(RoutingContext context, String templateFileName, Handler<Asyn
       }
       Map<String, RoutingContext> variables = new HashMap<>(1);
       variables.put("context", context);
-      handler.handle(Future.succeededFuture(Buffer.buffer((String)TemplateRuntime.execute(template, variables))));
+      String directoryName = Paths.get(templateFileName).getParent().toAbsolutePath().toString();
```

Comment:
Hi Paulo,
in the method where we are MVELTemplateEngineImpl.render() - we can't use a non static method from a static context (we can't do => File file1 = VertxInternal.resolveFile(templateFileName) 

PS My solution not work with jar files.

I will continue my investigation.